### ISSUE TITLE
Notify moves in MvInterface, etc.

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/pcdsdevices#528]
## Expected Behavior

Tell us which motor we're moving from where to where

## Current Behavior

N/A

## Possible Solution

```
@@ -389,6 +403,10 @@ class MvInterface(BaseInterface):
         super().__init__(*args, **kwargs)
         self._mov_ev = Event()

+    def _notify_move(self, position):
+        print('Moving {} from {} to {}'.format(self.name, self.wm(),
+                                               position))
+
     def mv(self, position, timeout=None, wait=False):
         """
         Absolute move to a position.
@@ -407,7 +425,7 @@ class MvInterface(BaseInterface):
             If `True`, wait for motion completion before returning.
             Defaults to :keyword:`False`.
         """
-
+        self._notify_move(position)
         self.move(position, timeout=timeout, wait=wait)

     def wm(self):
@@ -521,6 +539,7 @@ class FltMvInterface(MvInterface):
             will be use.
         """

+        self._notify_move(position)
         status = self.move(position, timeout=timeout, wait=False)
         AbsProgressBar([status])
         try:

```

## Context

Request from Tim

**Comment by ZLLentz** [comment]
Actually should use logger.info or self.log.info

**Comment by ZLLentz** [comment]
closed by #529
